### PR TITLE
added loader for shorten btn

### PR DESCRIPTION
**Pull request by sloorush**
solves issue #71

### CODE REVIEW

File: src/screens/Home.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import axios from 'axios';
 import Snackbar from '@material-ui/core/Snackbar';
 import Alert from '@material-ui/lab/Alert';
 import Grid from '@material-ui/core/Grid';
-import { Container } from '@material-ui/core';
+import { CircularProgress, Container } from '@material-ui/core';
```
Comment (QuanTumli) [inline src/screens/Home.js:7]:
```suggestion
import { Container } from '@material-ui/core';
```

File: src/screens/Home.js [hunk 2]
```diff
@@ -110,6 +110,9 @@ handleCopy = () => {
   };
 
   handleSubmit = async event => {
+    this.setState({
+      loading:true
```
Comment (QuanTumli) [inline src/screens/Home.js:114]:
```suggestion
      loading: true,
```

File: src/screens/Home.js [hunk 3]
```diff
@@ -192,6 +195,9 @@ handleCopy = () => {
         });
       }
     }
+    this.setState({
+      loading:false
```
Comment (QuanTumli) [inline src/screens/Home.js:199]:
```suggestion
      loading: false,
```

File: src/screens/Home.js [hunk 4]
```diff
@@ -274,8 +280,10 @@ handleCopy = () => {
                   cursor: 'pointer',
                   background: 'rgba(1, 87, 155, 100)',
                 }}
+                loading={this.state.loading}
                 onClick={this.handleSubmit}
-              ></MaterialButtonSuccess>
+              >                            
+</MaterialButtonSuccess>
```
Comment (QuanTumli) [inline src/screens/Home.js:286]:
```suggestion
              />
```

**Review by QuanTumli — CHANGES_REQUESTED** [review]

**Comment by r-ush** [comment]
Okay will do the required changes

File: src/screens/Home.js (hunk 1 above)
Comment (r-ush) [reply src/screens/Home.js:7]: Cannot remove CircularProgress as it is the component that has been called

**Comment by r-ush** [comment]
I have made the required changes @QuanTumli

**Comment by QuanTumli** [comment]
@r-ush Looks better :) Can you squash your commits into 1? See https://github.com/developer-student-club-thapar/urlShortner#commits

Travis-Checks are still failing: https://travis-ci.org/github/developer-student-club-thapar/urlShortner/jobs/731864814

It looks like there are not the correct indentation in `src/components/MaterialButtonSuccess.js` (too many spaces before your new code)

**Pull request closed without merging**